Fix `frontier/opcodes/test_all_opcodes.py::test_cover_revert` test

The test `tests/frontier/opcodes/test_all_opcodes.py::test_cover_revert[fork_Cancun-state_test]` fails with the following error

```
tests/frontier/opcodes/test_all_opcodes.py:126: in test_cover_revert
    state_test(env=Environment(), pre=pre, post={}, tx=tx)
src/pytest_plugins/execute/execute.py:400: in __init__
    execute.execute(fork=fork, eth_rpc=eth_rpc, engine_rpc=engine_rpc, request=request)
src/ethereum_test_execution/transaction_post.py:56: in execute
    eth_rpc.send_wait_transactions(signed_txs)
src/ethereum_test_rpc/rpc.py:356: in send_wait_transactions
    return self.wait_for_transactions(transactions)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
src/ethereum_test_rpc/rpc.py:329: in wait_for_transactions
    tx = self.get_transaction_by_hash(tx_hash)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
src/ethereum_test_rpc/rpc.py:230: in get_transaction_by_hash
    raise e
src/ethereum_test_rpc/rpc.py:225: in get_transaction_by_hash
    return TransactionByHashResponse.model_validate(
E   pydantic_core._pydantic_core.ValidationError: 1 validation error for TransactionByHashResponse
E   to_address
E     Field required [type=missing, input_value={'blockHash': '0x12f570e7... '0x1b', 'value': '0x0'}, input_type=dict]
E       For further information visit https://errors.pydantic.dev/2.11/v/missing

---------------------------- Captured stdout setup -----------------------------
Starting EOA index: 0xc01ae8b0557bc36d8038ea9c68c2f4f74a1059f412cc424a638221616fab8de5
Transaction hash 0x1c5ae4ef2329bfad3cc24cf006d9f9f02d74d98b8e9736204e00505400bb1bf0 does not match hash calculated from the RLP transaction data 0x7072642657de1929363a4a73e785e2787a258ff9c643cc0e336614b94b243bfe

----------------------------- Captured stdout call -----------------------------
Transaction hash 0x999e735de5e420095a006f3560feafef845ea35f347dcb70ef2d4598e31a8eab matches hash calculated from the RLP transaction data
[{'input': {'blockHash': '0x12f570e750a2f72e52c7217a86bb90d57160644a8207cf1bf9d38528a2731ebc',
            'blockNumber': '0x1884b',
            'from': '0x7ebcc520f137b54dbca52e59aea82c9b44db867f',
            'gas': '0xc3500',
            'gasPrice': '0xa54f4c3c00',
            'hash': '0x3664fc6b68433029b7dedb1e6b08cd815d6eca1245be990f43c5e068233f2d0d',
            'input': '0x600160015560006000fd',
            'nonce': '0x0',
            'r': '0x39f4963ce8e0a41c6c5bba9565d0bc308a68901fb30a6396a68e7631738d9d71',
            's': '0x5f02754ccc89382dc054183a7faa8b1c36db1dd1f354a1c1bf8d31d92bcf299f',
            'transactionIndex': '0x5',
            'type': '0x0',
            'v': '0x1b',
            'value': '0x0'},
  'loc': ('to_address',),
  'msg': 'Field required',
  'type': 'missing',
  'url': 'https://errors.pydantic.dev/2.11/v/missing'}]
--------------------------- Captured stdout teardown ---------------------------
Transaction hash 0x06a0da3974055eb8088d437ae59d170c427f348a0dd4408b9507a5d429ae71e6 does not match hash calculated from the RLP transaction data 0xccbbd7eaf193db8ef9bcec6186269d35150b2620a6101874241e23ed35905cfa
Used balance=0.597820000000000018
Transaction hash 0xf39ebaef6288591ed242751c88637bda029989c19e243e51002811e13453cc65 does not match hash calculated from the RLP transaction data 0x863248d19b486420efa7fc16d8ad1d3bf51ec204822e612ba25701235ef9e994
```

The root cause is that when a transaction sets `to:0x` and reverts, the JSON-RPC Relay does not include a `to` field in the response of `eth_getTransactionByHash`. Filed an issue in the JSON-RPC Relay

- https://github.com/hiero-ledger/hiero-json-rpc-relay/issues/4413

--- a/src/cli/pytest_commands/pytest_ini_files/pytest-execute.ini
+++ b/src/cli/pytest_commands/pytest_ini_files/pytest-execute.ini
@@ -33,7 +33,6 @@ addopts =
     # tests/frontier/identity_precompile/test_identity.py
     #
     # - Precompiles receiving HBARs https://github.com/gkozyryatskyy/execution-spec-tests/issues/12
-    # - Transaction sets `to:0x` and reverts, Relay doesn't have a `to` field in the response of `eth_getTransactionByHash` https://github.com/gkozyryatskyy/execution-spec-tests/issues/4
     # tests/frontier/opcodes/test_all_opcodes.py
     #
     # - Uses `BLOCKHASH(0)` which is not supported in long-lived networks

--- a/tests/frontier/opcodes/test_all_opcodes.py
+++ b/tests/frontier/opcodes/test_all_opcodes.py
@@ -54,7 +54,7 @@ def prepare_suffix(opcode: Opcode) -> Bytecode:
     pr=["https://github.com/ethereum/execution-spec-tests/pull/748"],
 )
 @pytest.mark.valid_from("Frontier")
-@pytest.mark.xfail(reason="https://github.com/gkozyryatskyy/execution-spec-tests/issues/12")
+@pytest.mark.skip(reason="https://github.com/gkozyryatskyy/execution-spec-tests/issues/12")
 def test_all_opcodes(state_test: StateTestFiller, pre: Alloc, fork: Fork):
     """
     Test each possible opcode on the fork with a single contract that calls
@@ -111,7 +111,6 @@ def test_all_opcodes(state_test: StateTestFiller, pre: Alloc, fork: Fork):
 
 
 @pytest.mark.valid_from("Cancun")
-@pytest.mark.xfail(reason="https://github.com/gkozyryatskyy/execution-spec-tests/issues/4")
 def test_cover_revert(state_test: StateTestFiller, pre: Alloc):
     """Cover state revert from original tests for the coverage script."""
     tx = Transaction(

